Streetcode/Streetcode.XUnitTest/MediatRTests/Media/Video/GetAllVideosTest.cs

```diff
@@ -0,0 +1,115 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore.Query;
+using Moq;
+using Streetcode.BLL.DTO.Media;
+using Streetcode.BLL.MediatR.Media.Video.GetAll;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Streetcode.XUnitTest.MediatRTests.Media.Video;
+
+public class GetAllVideosTest
+{
+    private Mock<IRepositoryWrapper> _mockRepository;
+    private Mock<IMapper> _mockMapper;
+
+    public GetAllVideosTest()
+    {
+        _mockRepository = new Mock<IRepositoryWrapper>();
+        _mockMapper = new Mock<IMapper>();
+    }
+
+    [Fact]
+    public async Task GetAllFacts_ShouldReturnSuccessfullyType()
```

Comment:
Why the name is GetAllFacts if we return videos?
If you specified the test method name in class name we don't need to repeat it here
Please consider using some naming conventions here

Reply:
Got it.